.github/workflows/docker-e2e.yml

```
@@ -0,0 +1,65 @@
+# This workflow pushes new osmosis-e2e-init-chain docker images on every new tag.
+#
+# On every new `vX.Y.0` tag the following images are pushed:
+#
+# osmolabs/osmosis-e2e-init-chain:X.Y.0    # is pushed
+# osmolabs/osmosis-e2e-init-chain:X.Y      # is updated to X.Y.0
+# osmolabs/osmosis-e2e-init-chain:X        # is updated to X.Y.0
+#
+# All the images above have support for linux/amd64 and linux/arm64.
+#
+# Due to QEMU virtualization used to build multi-platform docker images
+# this workflow might take a while to complete.
+name: Build and Push E2E Init Chain Images
+
+on:
+  push:
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+' # ignore rc
```

```suggestion
    - 'v[0-9]+.[0-9]+.0' # ignore rc
```

Wait the last one should be 0, isn't it? As per my earlier comment with test cases linked

---

I think it should be 0 too. Your last comment suggested changing to [ <> ](https://github.com/osmosis-labs/osmosis/blob/8ff4c7fe52657319f6b3c299c7002f7aa90d2d7b/.github/workflows/docker.yml#L27) 
so I did